auth/store.go

```diff
@@ -807,20 +879,53 @@ func (as *authStore) isAuthEnabled() bool {
 	return as.enabled
 }
 
-func NewAuthStore(be backend.Backend) *authStore {
+func NewAuthStore(be backend.Backend, tokenType, jwtSignMethod, jwtPubKeyPath, jwtPrivKeyPath string) *authStore {
```

Comment:
the problem is that it dose not really make sense for simpleToken to pass in the additional args.